homeassistant/components/pulseaudio_loopback/switch.py

```diff
@@ -186,6 +198,11 @@ def turn_off(self, **kwargs):
     def update(self):
         """Refresh state in case an alternate process modified this data."""
         self._pa_svr.update_module_state()
+
+        if not self._pa_svr.state_valid:
+            return
```

Comment:
Why do we return here?

Reply:
It doesn't make much sense to check if the loopback module for this switch is loaded when we already know we didn't receive any response...

Reply:
Okay thanks for the explanation.